src/app/interfaces/api/api-call-directory.interface.ts

```diff
@@ -318,13 +312,16 @@ export interface ApiCallDirectory {
   'app.latest': { params: QueryParams<AvailableApp>; response: AvailableApp[] };
   'app.similar': { params: [app_name: string, train: string]; response: AvailableApp[] };
   'app.rollback_versions': { params: [app_name: string]; response: string[] };
-  'app.used_ports': { params: void; response: number[] };
+
+  // App Image
+  'app.image.delete': { params: DeleteContainerImageParams; response: boolean };
+  'app.image.dockerhub_rate_limit': { params: void; response: DockerHubRateLimit };
+  'app.image.query': { params: QueryParams<ContainerImage>; response: ContainerImage[] };
 
   // Audit
   'audit.config': { params: void; response: AuditConfig };
   'audit.query': { params: [AuditQueryParams]; response: AuditEntry[] };
   'audit.update': { params: [AuditConfig]; response: AuditEntry[] };
-  'audit.download_report': { params: [{ report_name?: string }]; response: string[] };
 
```

Comment:
@themylogin Removes some more unused endpoints.

Reply:
Thank you! I created a JIRA ticket https://ixsystems.atlassian.net/browse/NAS-131853